<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESP8266 LuaLoader</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="file_tree_view">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tab_view">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="menu_file_new"/>
    <addaction name="menu_file_open"/>
    <addaction name="separator"/>
    <addaction name="menu_file_save"/>
    <addaction name="menu_file_save_as"/>
    <addaction name="separator"/>
    <addaction name="menu_file_close"/>
    <addaction name="separator"/>
    <addaction name="menu_file_quit"/>
   </widget>
   <widget class="QMenu" name="menu_settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="menu_preferences"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="menu_about"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="menu_view_actions"/>
    <addaction name="menu_view_files"/>
    <addaction name="separator"/>
    <addaction name="menu_view_log"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Delete"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_settings"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_file_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="menu_preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="action_Log">
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
  <action name="menu_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="menu_view_files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Files</string>
   </property>
  </action>
  <action name="menu_view_actions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Actions</string>
   </property>
  </action>
  <action name="action_GPIO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;GPIO</string>
   </property>
  </action>
  <action name="action_Wireless">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wireless</string>
   </property>
  </action>
  <action name="menu_view_log">
   <property name="text">
    <string>&amp;Log</string>
   </property>
  </action>
  <action name="menu_file_new">
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="menu_file_open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="menu_file_save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="menu_file_save_as">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="menu_file_close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
